spring-cloud-dataflow-core/src/test/java/org/springframework/cloud/dataflow/core/dsl/StreamParserTests.java

```diff
@@ -451,10 +432,10 @@ public void errorCases06() {
 
 	@Test
 	public void errorCases07() {
-		checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "bar");
-		checkForParseError("foo >", DSLMessage.OOD, 5);
-		checkForParseError("foo > --2323", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "--");
-		checkForParseError("foo > *", DSLMessage.UNEXPECTED_DATA, 6, "*");
+		//checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX, 6, "bar");
```

Comment:
can we not have a test that matches this commented-out one? (if not, we should delete rather than comment out, but it seems like we should ensure such a test does lead to an error still)